src/pkgs/files/login-inner.nix

```diff
@@ -16,12 +16,32 @@ writeScript "login-inner" ''
   shift
 
   [ "$#" -gt 0 ] || echo "Sourcing Nix environment..."
-  . ${instDir}/${packageInfo.nix}/etc/profile.d/nix.sh
+  . ${packageInfo.nix}/etc/profile.d/nix.sh
 
-  if [ ! -e $HOME/.nix-profile/etc/ssl/certs/ca-bundle.crt ]; then
-    if [ -e ${packageInfo.cacert} ]; then
-      export NIX_SSL_CERT_FILE=${packageInfo.cacert}
-    fi
+  if [ -e ${instDir}/etc/UNINTIALISED ]; then
+    export NIX_SSL_CERT_FILE=${packageInfo.cacert}
+
+    echo "Installing and updating nix-channels..."
+    ${packageInfo.nix}/bin/nix-channel --add https://nixos.org/channels/nixos-19.03 nixpkgs
+    ${packageInfo.nix}/bin/nix-channel --add https://github.com/t184256/nix-on-droid-bootstrap/archive/master.tar.gz nix-on-droid
```

Comment:
I envisioned it as a separate repository for some reason. But it can always be extracted later.

Reply:
I think it should be in the same repo as the other config files, because otherwise you would need to duplicate scripts like `bin/login` in to separate repositories.